src/pybind/mgr/dashboard/settings.py

```diff
@@ -57,6 +57,9 @@ class Options(object):
     ENABLE_BROWSABLE_API = Setting(True, [bool])
     REST_REQUESTS_TIMEOUT = Setting(45, [int])
 
+    # Wizard cluster creation status
+    POST_INSTALLED = Setting(False, [bool])
```

Comment:
Rather than a boolean, I'd use an enum-like `CLUSTER_STATUS` option here. Otherwise, we'll need to create options for every possible states, and handle the 'atomic' toggling of all of them...

Reply:
Done.